python/MOCK_sensors.py

```diff
@@ -142,8 +142,56 @@ def publish(self):
         # data.gyroscope_y_velocity_millidegreesps
         # data.gyroscope_z_velocity_millidegreesps
 
+        if self.sensorNoise:
```

Comment:
Right now, just looks at sensor_noise param from beginning. It would be nice if mid sim, we could run `rosparam set sensor_noise false` and it turn off.

`if getSensorNoise():`